support/build.gradle

```diff
@@ -1,8 +1,20 @@
+def VERSION_NAME = '0.1.0'
```

Comment:
Yes, I figured this needed to be bumped whenever we wanted to make a release with changes to the support library, which is why I wanted it at the top of the file to easily find.